compiler/damlc/daml-stdlib-src/DA/Action/State/Class.daml

```diff
@@ -10,11 +10,11 @@ module DA.Action.State.Class
 -- | Action `m` has a state variable of type `s`.
 --
 -- Rules:
---   get *> ma  =  ma
---   ma <* get  =  ma
---   put a >>= get   =  put a $> a
---   put a *> put b  =  put b
---   (,) <$> get <*> get  =  get <&> \a -> (a, a)
+--   * get *> ma  =  ma
```

Comment:
```suggestion
--   * `get *> ma  =  ma`
```
Does that make sense?

Reply:
I have no idea. I think it would get blue highlight then, not sure if this would be intended:
https://docs.daml.com/daml/stdlib/DA-Action-State-Class.html#class-da-action-state-class-actionstate-79096

Just please let me know how you prefer it.

Reply:
It’s code so wrapping it in a code block seems appropriate so if you don’t feel strongly about this, let’s add the backticks.

Reply:
Sure. I rebuilt the docs, looks good to me:
http://rst.ninjs.org/#ICAqIGBgZ2V0ICo+IG1hICA9ICBtYWBgCiAgKiBgYG1hIDwqIGdldCAgPSAgbWFgYAogICogYGBwdXQgYSA+Pj0gZ2V0ICAgPSAgcHV0IGEgJD4gYWBgCiAgKiBgYHB1dCBhICo+IHB1dCBiICA9ICBwdXQgYmBgCiAgKiBgYCgsKSA8JD4gZ2V0IDwqPiBnZXQgID0gIGdldCA8Jj4gXGEgLT4gKGEsIGEpYGAK